_posts/2020-09-17-using-tettra.md

```diff
@@ -0,0 +1,86 @@
+
+<!--more-->
+
+## Getting Started
+
+To make sure every team member knows how to navigate and use Tettra, we have a Using Tettra category. Under this category, we have created content to help team members get started with the tool, the integrations available and to get familiar with the structure of the knowledge base and how to navigate it.
+
+Here we have a style guide to explain how we write new pages, the documented structure of the knowledge base so every team member knows what each category is comprised of and where to find documents or add new content. This also includes our standard procedures to add new content or suggest new categories.
+
+## Knowledge Base Structure
+
+To keep our content organized, we use Tettra's categories and folders structure. Each category represents an area and each folder a division within that area. For example, under the Sales category, we have the Pre-Sales and Marketing folders.
+
+We also use Folders to keep our Draft pages before they are officially published to the whole team.
+
+<img src="/blog/assets/images/tettra/tettra_categories.jpg" alt="tettra categories" class="medium-img">
+
+## Adding Content
+
+Tettra allows you to create pages inside the tool as well as add a link to external documents (Google Doc or Google Sheet). This lets us create new pages as needed as well as link to our documents available through Google Docs and Google Sheets, keeping all the information in a single place.
+
+To make sure we keep things organized and standardized, we created a style guide for our Tettra pages that can be found under the Using Tettra category. Additionally, we also have a standard procedure for adding content.
+
+Tettra allows users to either add new content in the form of a Tettra page, link to an external document (Google Doc or Google Sheet) or make a suggestion.
+
+When a team member wants to add content to the knowledge base, they can simply create a new page and save it to the Drafts folder of the category they want to add the page to. Once the page is published under the Drafts folder, they can use the Comments feature to request a review from other team members. Once the page has been reviewed and approved, it is moved to the proper Folder under the Category and made available to the team.
+
+However, sometimes a team member wants to see something in the knowledge base but is not sure what's the best way to add it. In that case, they can create a suggestion and either assign it to the best team member to write a page or to the knowledge base admin, who'll make sure the suggestion is addressed.
```

Comment:
@ABizzinotto In this paragraph it seems that we are missing a step. Maybe you can mention that when they want to find or see something they can easily search through all Tettra pages and create a suggestion for an existing page? I think the searchability of the KB is the most important feature. :)

Reply:
Ah! Great point 👍 I'll update it.

Reply:
Actually, I ended up adding a new section about searching for content :)